### PR TITLE
Add a new RPC and Cap to advertise the client details

### PR DESCRIPTION
**Pull request by Madhu-1**
added a GET_CLIENTS_TO_FENCE so that the plugins can register with these caps to advertise that they support publishing the metadata like identifier and the cluster IP addresses of the clone cline for the fence operations.

Adding an RPC to get the local client details which includes the id and the local IP addresses for fencing. The ID can be anything that can be used as an identifier of the client address. For example, the ID can be the cephcluster UUID and the ip address/cidr is the local client address used to connect to the ceph cluster, The ceph is an example/representation here.

### CODE REVIEW

File: identity/README.md [hunk 1]
```diff
@@ -234,6 +234,13 @@ message Capability {
       // plugin can invoke RPCs that require access to the storage system,
       // similar to the CSI Controller (provisioner).
       NETWORK_FENCE = 1;
+
+      // GET_CLIENTS_TO_FENCE indicates that the CSI-driver provides RPCs for a
+      // GET_CLIENTS_TO_FENCE operation to fet the clients to fence.
```
Comment (nixpanic) [inline identity/README.md:239]: typo _fet_

Also a typo _cline_ in the commit message.

File: fence/README.md [hunk 2]
```diff
@@ -66,6 +66,11 @@ service FenceController {
   // ListClusterFence RPC call to provide a list of blocklisted/fenced clients.
   rpc ListClusterFence(ListClusterFenceRequest)
   returns (ListClusterFenceResponse){}
+
+  // GetFenceClients RPC calls to get the clients information that needs
+  // to be fenced.
```
Comment (nixpanic) [inline fence/README.md:71]: Maybe _.. get the client information to use in a FenceClusterNetwork or UnfenceClusterNetwork RPC._

**Review by nixpanic — CHANGES_REQUESTED** [review]
Looks reasonable to me. Just need some typo fixes, and possibly rephrasing the description of the new RPC a bit.

File: identity/README.md [hunk 3]
```diff
@@ -234,6 +234,13 @@ message Capability {
       // plugin can invoke RPCs that require access to the storage system,
       // similar to the CSI Controller (provisioner).
       NETWORK_FENCE = 1;
+
+      // GET_CLIENTS_TO_FENCE indicates that the CSI-driver provides RPCs for a
+      // GET_CLIENTS_TO_FENCE operation to get the clients to fence.
+      // The presence of this capability determines whether the CSI-Addons CO
+      // plugin can invoke RPCs that require access to the storage system,
+      // similar to the CSI Controller (provisioner).
```
Comment (nixpanic) [inline identity/README.md:242]: Thanks for fixing the typos, however there is now _clinets_ in the commit description :laughing:

**Review by nixpanic — DISMISSED** [review]

File: identity/README.md (hunk 3 above)
Comment (Madhu-1) [reply identity/README.md:242]: Addressed now 🤦🏻

**Review by nixpanic — APPROVED** [review]

**Pull request merged**